utils/env.go

```diff
@@ -3,6 +3,7 @@ package utils
 import (
```

Comment:
I like how this whole file is evolving. Good tooling which probably might be replaced by viper someday.